tests/integration/test_ses.py

```diff
@@ -309,3 +309,148 @@ def test_ses_sns_topic_integration(
+            "$..Message.delivery.processingTimeMillis",
+            "$..Message.delivery.reportingMTA",
+            "$..Message.delivery.smtpResponse",
+            "$..Message.mail.messageId",
+            "$..Message.mail.commonHeaders",
+            "$..Message.mail.headers",
+            "$..Message.mail.headersTruncated",
+            "$..Message.mail.tags",
+            "$..Message.mail.timestamp",
+        ]
+    )
+    def test_ses_sns_topic_integration_send_templated_email(
+        self,
+        ses_client,
+        ses_configuration_set,
+        ses_configuration_set_sns_event_destination,
+        ses_email_template,
+        ses_verify_identity,
+        sns_create_sqs_subscription,
+        sns_create_topic,
+        sqs_queue,
+        sqs_receive_num_messages,
+        snapshot,
+    ):
+        sender_email_address = f"repro-7184-{short_uid()}@example.com"
+        recipient_email_address = f"repro-7184-{short_uid()}@example.com"
+
+        snapshot.add_transformers_list(
+            [
+                snapshot.transform.regex(sender_email_address, "<sender-email-address>"),
+                snapshot.transform.regex(recipient_email_address, "<recipient-email-address>"),
+            ]
+            + snapshot.transform.sns_api()
+        )
+
+        ses_verify_identity(sender_email_address)
+        ses_verify_identity(recipient_email_address)
+
+        template_name = f"template-{short_uid()}"
+        ses_email_template(template_name, "Test template")
+
+        # create queue to listen for for SES -> SNS events
+        topic_arn = sns_create_topic()["TopicArn"]
+        sns_create_sqs_subscription(topic_arn=topic_arn, queue_url=sqs_queue)
+
+        # create the config set
+        config_set_name = f"config-set-{short_uid()}"
+        ses_configuration_set(config_set_name)
+        event_destination_name = f"config-set-event-destination-{short_uid()}"
+        ses_configuration_set_sns_event_destination(
+            config_set_name, event_destination_name, topic_arn
+        )
+
+        # send an email to trigger the SNS message and SQS message
+        destination = {
+            "ToAddresses": [recipient_email_address],
+        }
+        ses_client.send_templated_email(
+            Destination=destination,
+            Template=template_name,
+            TemplateData=json.dumps({}),
+            ConfigurationSetName=config_set_name,
+            Source=sender_email_address,
+        )
+
+        messages = sqs_receive_num_messages(sqs_queue, 3)
+        snapshot.match("messages", messages)
+
+    @pytest.mark.only_localstack
+    @pytest.mark.skip_snapshot_verify(
+        paths=[
+            "$..Signature",
+            "$..SigningCertURL",
+            "$..TopicArn",
+            "$..UnsubscribeURL",
+            "$..Message.delivery.processingTimeMillis",
+            "$..Message.delivery.reportingMTA",
+            "$..Message.delivery.smtpResponse",
+            "$..Message.mail.messageId",
```

Comment:
nit: as this is going to be different between each test run, to avoid committing the changes to the snapshot, it maybe could be transformed using adding a transformer to the list:
`snapshot.transform.key_value("messageId")`, it would keep the reference and try to replace it across the test.

On a more global note, I tend to think about the `pytest.mark.skip_snapshot_verify` as something we don't implement yet and kind of a todo list, and to transform things that we can't control (generated ids, timestamp). Sometimes, you have to use ignore for hard to replace things like the `UnsubscribeURL`, but the `TopicArn` could be transformed as well.

And sometimes, it's also hard because you would like to keep part of the response and not replace everything. I guess `skip_snapshot_verify` can still be used in that case. Anyway, this was more of a philosophical comment!

Reply:
You're 100% right with with the message id there (and a few other properties). The original issue reporter said they needed that field as part of their output, and I didn't realise it was deterministic (i.e. associated with the email and not some UUID). I corrected this as part of #7244. Most of the other properties cannot really be verified (and likely don't provide any additional value) so I feel we can probably skip verification of them. Thanks for the feedback!

I've removed any properties the `sns` transformer transforms (so we're at least verifying their presence and approximate contents).

Reply:
Amazing, thanks a lot!